### PR TITLE
Jason chatbot

### PR DESCRIPTION
**Pull request by Topia99**

### CODE REVIEW

File: Travel-Planner/Travel-Planner/Screens/ChatView.swift [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    var body: some View {
+        VStack {
+            ScrollView {
+                ScrollViewReader { scrollView in
+                    LazyVStack(alignment: .leading, spacing: 12) {
+                        ForEach(vm.messages) { message in
+                            MessageView(message: message)
+                                .padding(.horizontal)
+                        }
+//                            Spacer(minLength: 500) // add extra space below
```
Comment (Topia99) [inline Travel-Planner/Travel-Planner/Screens/ChatView.swift]: remove

File: Travel-Planner/Travel-Planner/Views/DayPlanView/DateCapsuleView.swift [hunk 2]
```diff
@@ -13,14 +13,40 @@ struct DateCapsuleView: View {
...
+    
+        
+        
+    
+//        Text(isSelected ? DateUtils.formattedDate_WeekDay_Date(date) : DateUtils.formattedDate(date))
+//            .font(.system(size: 18))
+//            .fontWeight(.semibold)
+//            .frame(width: isSelected ? 120 : 90, height: 35)
+//            .background(isSelected ? Color.brandPrimary : BlurView(style: .systemMaterial))
+//            .foregroundColor(.white)
```
Comment (Topia99) [inline Travel-Planner/Travel-Planner/Views/DayPlanView/DateCapsuleView.swift]: remove

File: Travel-Planner/Travel-Planner/Views/DayPlanView/DayPlan/DayPlanHeadingView.swift [hunk 3]
```diff
@@ -16,10 +16,11 @@ struct DayPlanHeadingView: View {
         // DayPlan Date
         HStack(alignment: .firstTextBaseline) {
             Text(DateUtils.formattedDate(date))
+                .foregroundColor(Color.white)
                 .font(.title)
                 .fontWeight(.semibold)
             
-            LabelledDivider(label: "day \(dayNumber)")
```
Comment (Topia99) [inline Travel-Planner/Travel-Planner/Views/DayPlanView/DayPlan/DayPlanHeadingView.swift:22]: remove

File: Travel-Planner/Travel-Planner/Views/DayPlanView/DayPlan/DayPlanView.swift [hunk 4]
```diff
@@ -16,35 +16,46 @@ struct DayPlanView: View {
...
-                // Add new activity button
-                Button {
-                    isShowingAddActivity = true
-                } label: {
-                    AddItemButton()
-                }
-                
+                .listRowSeparator(.hidden)
+                .listRowInsets(.init(top: 0, leading: 0, bottom: 0, trailing: 0))
+//                .listRowBackground(Color.clear)
```
Comment (Topia99) [inline Travel-Planner/Travel-Planner/Views/DayPlanView/DayPlan/DayPlanView.swift]: remove

**Review by Topia99 — COMMENTED** [review]
good

**Review by Topia99 — COMMENTED** [review]
good to merge

**Review by SamYang10086 — COMMENTED** [review]
pass

**Review by SamYang10086 — APPROVED** [review]
pass

**Pull request merged**